src/test/fuzz/addrman.cpp

```diff
@@ -275,7 +275,16 @@ FUZZ_TARGET(addrman, .init = initialize_addrman)
         /*max_pct=*/fuzzed_data_provider.ConsumeIntegralInRange<size_t>(0, 4096),
         network,
         /*filtered=*/fuzzed_data_provider.ConsumeBool());
-    (void)const_addr_man.Select(fuzzed_data_provider.ConsumeBool(), network);
+
+    auto all_networks = ALL_NETWORKS;
+    FastRandomContext fast_random_context{ConsumeUInt256(fuzzed_data_provider)};
+    Shuffle(all_networks.begin(), all_networks.end(), fast_random_context);
+    std::vector<Network> nets;
+    for (size_t i = 0; i < all_networks.size() && fuzzed_data_provider.ConsumeBool(); i++) {
+        nets.push_back(all_networks[i]);
+    }
+    (void)const_addr_man.Select(fuzzed_data_provider.ConsumeBool(), nets);
```

Comment:
This could end up passing an empty container (`vector` or `unordered_set` if you decide to change it). Maybe this:

```cpp
if (nets.empty()) {
    (void)const_addr_man.Select(fuzzed_data_provider.ConsumeBool());
} else {
    (void)const_addr_man.Select(fuzzed_data_provider.ConsumeBool(), nets);
}

Reply:
Empty container and nothing means the samething, all networks.